docs/incremental_processing.md

```diff
@@ -4,134 +4,135 @@ keywords: incremental processing
-    --target-table
-       name of the target table in Hive
+      Field within source record to decide how to break ties between records
+      with same key in input data. Default: 'ts' holding unix timestamp of
+      record
+      Default: ts
+    --spark-master
+      spark master to use.
+      Default: local[2]
+  * --target-base-path
+      base path for the target hoodie dataset. (Will be created if did not
+      exist first time around. If exists, expected to be a hoodie dataset)
+  * --target-table
+      name of the target table in Hive
 
 
 ```
 
-For e.g, followings ingests data from Kafka (avro records as the client example)
 
+The tool takes a hierarchically composed property file and has pluggable interfaces for extracting data, key generation and providing schema. Sample configs for ingesting from kafka and dfs are
+provided under `hoodie-utilities/src/test/resources/delta-streamer-config`.
+
+For e.g: once you have Confluent Kafka, Schema registry up & running, produce some test data using ([impressions.avro](https://docs.confluent.io/current/ksql/docs/tutorials/generate-custom-test-data.html) provided by schema-registry repo)
 
 ```
-[hoodie]$ spark-submit --class com.uber.hoodie.utilities.deltastreamer.HoodieDeltaStreamer hoodie-utilities/target/hoodie-utilities-0.3.6-SNAPSHOT-bin.jar \
-     --hoodie-client-config hoodie-utilities/src/main/resources/delta-streamer-config/hoodie-client.properties \
-     --key-generator-config hoodie-utilities/src/main/resources/delta-streamer-config/key-generator.properties \
-      --schemaprovider-config hoodie-utilities/src/main/resources/delta-streamer-config/schema-provider.properties \
-      --source-class com.uber.hoodie.utilities.sources.KafkaSource \
-      --source-config hoodie-utilities/src/main/resources/delta-streamer-config/source.properties \
-      --source-ordering-field rider \
-      --target-base-path file:///tmp/hoodie-deltastreamer-op \
-      --target-table uber.trips
+[confluent-5.0.0]$ bin/ksql-datagen schema=../impressions.avro format=avro topic=impressions key=impressionid
 ```
 
+and then ingest it as follows.
+
+```
+[hoodie]$ spark-submit --class com.uber.hoodie.utilities.deltastreamer.HoodieDeltaStreamer `ls hoodie-utilities/target/hoodie-utilities-*-SNAPSHOT.jar` \
+  --props file://${PWD}/hoodie-utilities/src/test/resources/delta-streamer-config/kafka-source.properties \
+  --schemaprovider-class com.uber.hoodie.utilities.schema.SchemaRegistryProvider \
+  --source-class com.uber.hoodie.utilities.sources.AvroKafkaSource \
+  --source-ordering-field impresssiontime \
+  --target-base-path file:///tmp/hoodie-deltastreamer-op --target-table uber.impressions \
+  --op BULK_INSERT
+```
+
+In some cases, you may want to convert your existing dataset into Hoodie, before you can begin ingesting new data. This can be accomplished using the `hdfsparquetimport` command on the `hoodie-cli`.
+Currently, there is support for converting parquet datasets.
```

Comment:
Do we want to say, "Currently, there is support for converting **only** parquet datasets" ?